### ISSUE TITLE
Add the ability to specify which types should trigger a notification in the same way types can be specified on cameras

### ISSUE THREAD

**Issue by Martinnygaard** [djdd87/SynoAI#28]
Hi,

I would find it useful if I can have it send different webhook notifications towards homeassistant whether it is a car or a person it has detected. 
Or even better, if I somehow can load the detected object as a variable. (I don't know the limitation of webhooks), but if there could be a payload, telling what objects are detected (eg. 2 cars and 1 person).

Will this be possible?

**Comment by djdd87** [comment]
Hi @Martinnygaard, yes it's perfectly possible to include variables in web hooks. This is actually already done for POST requests as it'll attach the array as a JSON list of types as well as the image. Note that the GET method doesn't support a payload by the design of the HTTP standards, so any variables must exist in the query parameters of the URL - this isn't something that's currently included.

The simplest thing for being able to send to different webhooks based on the type would be for me to add a "Types" property to Notifications as well. We already have this for the cameras, so my proposed solution would mean you have:
```json
  "Notifiers": [
    {
      "Type": "Webhook",
      "Url": "http://server/images",
      "Types": ["Person"]
    },
    {
      "Type": "Webhook",
      "Url": "http://server/images",
      "Types": ["Car"]
    }
  ],

  "Cameras": [
    {
      "Name": "Driveway",
      "Types": [ "Person", "Car" ],
      "Threshold": 45
    }
  ]
```

**Comment by Martinnygaard** [comment]
Nope, that is acetly how i first thought about it myself. I will give it a try! :-)

**Comment by djdd87** [comment]
Well, you can't try it yet as I haven't written the code yet ;). But I can add that if that'll solve your problem.

**Comment by Martinnygaard** [comment]
Ahh, you are properly right then ;-P
Just mis-read your post.. Lol..

Yes, it will definitely help my need!

**Comment by JohannCR** [comment]
That'd be neat, is that feature still in the works ?
Thanks

**Comment by JohannCR** [comment]
In the same vein, is it possible to make notifications conditional on sss not being in home mode ?
Or is there another way to do that ?

**Comment by djdd87** [comment]
@JohannCR can you request that separately please as it's outside the scope of this request.

**Comment by JohannCR** [comment]
@djdd87 not sure about that after thinking about it a bit. What I do for the moment is deactivate all sss action rules when in home mode, while recording on every movement. I think it's preferable than only deactivating synoAI notifications, particularly on situation where I know there will be lots of movements in front of the cameras. What do you think ?

On the other hand, being able to send notifications for specific types, while only using webhooks for others for example is way more powerful and useful... Right now I have to run two instances of synoAI, one that detects persons and sends notifications, and another one that detects cats/dogs and only triggers a webhook... It's a bit messy with the action rules ^^
This here enhancement seems more important (like the "perfect shot" one).

Thanks for your work, really happy with synoai !

**Comment by djdd87** [comment]
I'll _try_ and have a look at this feature request tonight.

**Comment by EzequielPitton78** [comment]
Just for your consideration: for a similar need I process the sss action rules in node red, so the  logic to apply or not synoAI is done there. Th appropriate way  o react to the detection events is also done there. That way you can apply whatever complex logic you need before and/or after.

**Comment by lucasCFO** [comment]
@EzequielPitton78 could you share a flow example of your work pls?

**Comment by EzequielPitton78** [comment]
Of course! Below you have the flows (sorry, descriptions in spanish). 

The first part is called by SSS action rule. The action rule is configured with a parameter camara=xxxx so you can know what camera is calling.  (action rule webhook format -> http://192.168.1.5:1880/movimiento_camara?camara=xxx) and calls SynoAI conditionally. In this example, if a global variable is true.

The second part is called by SynoAI notifier, where I also read the camera by parameter. In this example I just update a mqtt topic.

      {
        "Type": "Webhook",
        "Cameras": ["HDW4631C-A-PATIO"],
        "Url": "http://192.168.1.5:1880/deteccion_ai?camara=patio",
        "Method": "POST",
        "Field": "image",
        "SendImage": false,
        "SendTypes": true
      },




[{"id":"aaab4a3fe4f81e83","type":"tab","label":"Camaras","disabled":false,"info":""},{"id":"b4eae41e1256c5a2","type":"switch","z":"aaab4a3fe4f81e83","name":"Movimiento Detectado? ","property":"payload.camara","propertyType":"msg","rules":[{"t":"eq","v":"patio","vt":"str"},{"t":"eq","v":"garage","vt":"str"},{"t":"eq","v":"fondo","vt":"str"},{"t":"eq","v":"despacho_papa","vt":"str"},{"t":"eq","v":"barbacoa","vt":"str"},{"t":"eq","v":"jardin_piedra","vt":"str"},{"t":"eq","v":"trastero","vt":"str"}],"checkall":"false","repair":false,"outputs":7,"x":590,"y":260,"wires":[["0b843185a5415fcb"],["b723293374bf1abf"],["6b1c634b4932e745"],["49e54fd3bcf74e99"],["72352ff2622fa246"],["70deb920c3a76fbd"],["5a68526c7974e645"]]},{"id":"25219a9e958b0080","type":"http in","z":"aaab4a3fe4f81e83","name":"Movimiento Camara","url":"/movimiento_camara","method":"get","upload":false,"swaggerDoc":"","x":270,"y":260,"wires":[["b4eae41e1256c5a2","7fbea2925066850a"]]},{"id":"7fbea2925066850a","type":"http response","z":"aaab4a3fe4f81e83","name":"Respuesta","statusCode":"200","headers":{},"x":350,"y":360,"wires":[]},{"id":"5bc3146a82fdc21b","type":"mqtt out","z":"aaab4a3fe4f81e83","name":"Guardar Estado MQTT","topic":"","qos":"","retain":"","broker":"fe277c831b539ac1","x":1780,"y":420,"wires":[]},{"id":"0b843185a5415fcb","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Patio","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-PATIO","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-PATIO","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/patio/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":980,"y":140,"wires":[["7b0785f4c150c3fc"]]},{"id":"76838c6408d190e1","type":"http request","z":"aaab4a3fe4f81e83","name":"Llamar a SynoAI","method":"GET","ret":"txt","paytoqs":"ignore","url":"","tls":"","persist":false,"proxy":"","authType":"","x":2050,"y":140,"wires":[[]]},{"id":"49e54fd3bcf74e99","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Despacho Papa","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-DESPACHO_PAPA","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-DESPACHO_PAPA","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/despacho_papa/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":1010,"y":260,"wires":[["85d7d07fdf0e606e"]]},{"id":"85d7d07fdf0e606e","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1340,"y":260,"wires":[["5bc3146a82fdc21b","57e59e5fd25470cc"]]},{"id":"7b0785f4c150c3fc","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1340,"y":140,"wires":[["5bc3146a82fdc21b","57e59e5fd25470cc"]]},{"id":"b723293374bf1abf","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Garage","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-GARAGE","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-GARAGE","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/garage/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":980,"y":180,"wires":[["0b57126ff62f4c8b"]]},{"id":"0b57126ff62f4c8b","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1340,"y":180,"wires":[["5bc3146a82fdc21b","57e59e5fd25470cc"]]},{"id":"6b1c634b4932e745","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Fondo","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-FONDO","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-FONDO","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/fondo/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":980,"y":220,"wires":[["e0a6c1e037de8d73"]]},{"id":"e0a6c1e037de8d73","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1340,"y":220,"wires":[["5bc3146a82fdc21b","57e59e5fd25470cc"]]},{"id":"72352ff2622fa246","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Barbacoa","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-BARBACOA","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-BARBACOA","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/barbacoa/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":990,"y":300,"wires":[["cc1fcb03acc4adc3"]]},{"id":"cc1fcb03acc4adc3","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1340,"y":300,"wires":[["5bc3146a82fdc21b","57e59e5fd25470cc"]]},{"id":"70deb920c3a76fbd","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Jardin Piedra","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-JARDIN_PIEDRA","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-JARDIN_PIEDRA","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/jardin_piedra/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":1000,"y":340,"wires":[["e8e237374ae8cc4f"]]},{"id":"e8e237374ae8cc4f","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1340,"y":340,"wires":[["5bc3146a82fdc21b","57e59e5fd25470cc"]]},{"id":"5a68526c7974e645","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Trastero","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-TRASTERO","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-TRASTERO","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/trastero/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":990,"y":380,"wires":[["1c83722f813ee27e"]]},{"id":"1c83722f813ee27e","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1340,"y":380,"wires":[["5bc3146a82fdc21b","57e59e5fd25470cc"]]},{"id":"ebc2f9d2620b8a40","type":"comment","z":"aaab4a3fe4f81e83","name":"DETECCION MOVIMIENTO","info":"","x":180,"y":100,"wires":[]},{"id":"2f543741a14aa7a6","type":"switch","z":"aaab4a3fe4f81e83","name":"Objeto Detectado? ","property":"req.query.camara","propertyType":"msg","rules":[{"t":"eq","v":"patio","vt":"str"},{"t":"eq","v":"garage","vt":"str"},{"t":"eq","v":"fondo","vt":"str"},{"t":"eq","v":"despacho_papa","vt":"str"},{"t":"eq","v":"barbacoa","vt":"str"},{"t":"eq","v":"jardin_piedra","vt":"str"},{"t":"eq","v":"trastero","vt":"str"}],"checkall":"false","repair":false,"outputs":7,"x":570,"y":840,"wires":[["ce0376021305d594"],["ae835293d4ebd2af"],["ee69acf8ff4c759d"],["68c90970ae43194a"],["2650162b461bb8c6"],["3a228ba3229e555d"],["22e7444059c36b4a"]]},{"id":"8d35b1ab6aa9721c","type":"http in","z":"aaab4a3fe4f81e83","name":"Deteccion objetos","url":"/deteccion_ai","method":"post","upload":false,"swaggerDoc":"","x":250,"y":840,"wires":[["2f543741a14aa7a6","83ca4d4691a14f18"]]},{"id":"83ca4d4691a14f18","type":"http response","z":"aaab4a3fe4f81e83","name":"Respuesta","statusCode":"200","headers":{},"x":270,"y":940,"wires":[]},{"id":"49796dd1bb4ed5b3","type":"mqtt out","z":"aaab4a3fe4f81e83","name":"Guardar Estado MQTT","topic":"","qos":"","retain":"","broker":"fe277c831b539ac1","x":1780,"y":840,"wires":[]},{"id":"ce0376021305d594","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Patio","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-PATIO","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-PATIO","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/patio/persona","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":960,"y":700,"wires":[["1f294032bef15e8a"]]},{"id":"68c90970ae43194a","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Despacho Papa","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-DESPACHO_PAPA","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-DESPACHO_PAPA","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/despacho_papa/persona","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":990,"y":820,"wires":[["007b4b1ce37c1b8a"]]},{"id":"007b4b1ce37c1b8a","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1320,"y":820,"wires":[["49796dd1bb4ed5b3"]]},{"id":"1f294032bef15e8a","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1320,"y":700,"wires":[["49796dd1bb4ed5b3"]]},{"id":"ae835293d4ebd2af","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Garage","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-GARAGE","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-GARAGE","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/garage/movimiento","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":960,"y":740,"wires":[["7e80971df046f65a"]]},{"id":"7e80971df046f65a","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1320,"y":740,"wires":[["49796dd1bb4ed5b3"]]},{"id":"ee69acf8ff4c759d","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Fondo","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-FONDO","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-FONDO","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/fondo/persona","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":960,"y":780,"wires":[["da766522af5d7418"]]},{"id":"da766522af5d7418","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1320,"y":780,"wires":[["49796dd1bb4ed5b3"]]},{"id":"2650162b461bb8c6","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Barbacoa","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-BARBACOA","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-BARBACOA","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/barbacoa/persona","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":970,"y":860,"wires":[["b145611605752ce8"]]},{"id":"b145611605752ce8","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1320,"y":860,"wires":[["49796dd1bb4ed5b3"]]},{"id":"3a228ba3229e555d","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Jardin Piedra","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-JARDIN_PIEDRA","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-JARDIN_PIEDRA","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/jardin_piedra/persona","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":980,"y":900,"wires":[["7e65e48cb0940a6f"]]},{"id":"7e65e48cb0940a6f","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1320,"y":900,"wires":[["49796dd1bb4ed5b3"]]},{"id":"22e7444059c36b4a","type":"change","z":"aaab4a3fe4f81e83","name":"Camara Trastero","rules":[{"t":"set","p":"camara","pt":"msg","to":"HDW4631C-A-TRASTERO","tot":"str"},{"t":"set","p":"payload","pt":"msg","to":"true","tot":"bool"},{"t":"set","p":"url","pt":"msg","to":"http://192.168.1.5:8180/Camera/HDW4631C-A-TRASTERO","tot":"str"},{"t":"set","p":"topic","pt":"msg","to":"camaras/trastero/persona","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":970,"y":940,"wires":[["564c49895c13f3d4"]]},{"id":"564c49895c13f3d4","type":"trigger","z":"aaab4a3fe4f81e83","name":"Restaurar en 5 s","op1":"","op2":"false","op1type":"pay","op2type":"bool","duration":"5","extend":true,"overrideDelay":false,"units":"s","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":1320,"y":940,"wires":[["49796dd1bb4ed5b3"]]},{"id":"770b1ab78640ba47","type":"comment","z":"aaab4a3fe4f81e83","name":"DETECCION PRESENCIA","info":"","x":170,"y":620,"wires":[]},{"id":"57e59e5fd25470cc","type":"switch","z":"aaab4a3fe4f81e83","name":"Alarma Activada?","property":"alarma_activada","propertyType":"global","rules":[{"t":"true"}],"checkall":"true","repair":false,"outputs":1,"x":1770,"y":140,"wires":[["76838c6408d190e1"]]},{"id":"fe277c831b539ac1","type":"mqtt-broker","name":"CASA MQTT","broker":"192.168.1.5","port":"1883","clientid":"","usetls":false,"compatmode":true,"keepalive":"60","cleansession":true,"birthTopic":"","birthQos":"0","birthPayload":"","closeTopic":"","closeQos":"0","closePayload":"","willTopic":"","willQos":"0","willPayload":""}]

**Comment by EzequielPitton78** [comment]
Please bear in mind im not a programer, so the code is sure not as elegant as it could be (actually i have near 40 cameras, so is quite ugly) . 
Furthermore I have not been yet able to read the detection types from node red, maybe is not the right format. That's why I only set it up for person type. 

Many thanks @djdd87 BTW for your time puting this together :)  


Ezequiel

**Comment by djdd87** [comment]
You can now specify that notifications can only run when a matching type is detected by the AI, e.g. 

``` json
{
  "Type":"Telegram",
  "ChatID": "123",
  "Token": "XYZ",
  "Types": ["Person"]
}
```

**Comment by JohannCR** [comment]
Got a Pushbullet notification for a type outside of the "Types" list... Do you need anything to debug ?

**Comment by JohannCR** [comment]
Seems good now. Awesome

**Comment by djdd87** [comment]
@JohannCR - sorry didn't see this. I don't recall doing anything to cause that or stop that, is it sorted or has it happened again since?

**Comment by JohannCR** [comment]
@djdd87 ah ah ^^ weird. Must have been an error in the config then, I've modified it multiple times so I don't know what caused it... Cannot test it thoroughly right now but I will and report back. Thanks for everything

**Comment by djdd87** [comment]
Available in `v1.1.0`.
